generators/app/templates/docs/_standards.md

```diff
@@ -0,0 +1,126 @@
+`src/plugin.scss`          | ✓        | Sass entry point.
+`src/plugin.js`            |          | JavaScript entry point.
+`test/`                    |          | All testing-related source code.
+`test/karma/`              |          | Karma configuration files.
+`test/unit/`               |          | Unit tests.
+`test/unit/plugin.test.js` |          | Testing entry point.
+`.editorconfig`            |          |
+`.gitignore`               |          |
+`.npmignore`               |          | Override `.gitignore`'s exclusion of `dist/`.
+`bower.json`               |          |
+`CHANGELOG.md`             | ✓        | May be removed if not desired.
+`CONTRIBUTING.md`          | ✓        | Not present in closed-source plugins.
+`index.html`               |          | An example of usage of the plugin. This can be used with GitHub pages as well.
+`Gruntfile.js`             | ✓        |
+`LICENSE`                  | ✓        | Defaults to `MIT`.
+`package.json`             |          |
+`README.md`                |          | Documents which version(s) of video.js the plugin supports. Explains how to build/test.
+
+## Building
+
+__All tooling for a standard video.js plugin must be available through npm scripts.__
+
+There are many reasons for this rule:
+
+- Complicated build tools are not necessary for all plugins.
+- Where a separate build tool is preferred, npm scripts can easily act as aliases to build tasks (e.g., `"build": "grunt build"`).
+- CI servers and other tools can use npm - as they must anyway, in some capacity - without worrying about the underlying tooling.
+- Consistent npm script naming means contributors don't have to learn a new build tool or new set of commands when moving between plugin projects, lowering the barrier to contributions.
+
+### Grunt, by Default
+
+By default, a Grunt-based workflow is provided by the generator. Its tasks closely match the npm scripts outlined below. By default, it will run `grunt test`.
+
+### npm Scripts
+
+__The core set of npm scripts must match the table below.__
+
+All names are lower-case and use colons (`:`) as word-separators. Certain `pre*` and `post*` scripts will be created as well, but these are not documented here and are, therefore, subject to change.
+
+npm Script   | Grunt Equiv.       | Optional | Description
+------------ | ------------------ | -------- | -----------
+`build`      | `grunt build`      |          | Runs all build sub-tasks.
+`build:css`  | `grunt build:css`  | ✓        | Builds the Sass entry point.
+`build:js`   | `grunt build:js`   |          | Builds the Browserify entry point.
+`build:test` | `grunt build:test` |          | Builds the test Browserify entry point.
+`clean`      | `grunt clean`      |          | Cleans up build artifacts.
+`lint`       | `grunt lint`       |          | Lints all `.js` file(s).
+`start`      | `grunt start`      |          | Starts a development server and runs `watch`.
```

Comment:
I'm cool with noting the first part, but don't feel that we should document that `npm run` will list available scripts.